### PR TITLE
Update upgrading-always-on-availability-group-replica-instances.md

### PR DESCRIPTION
**Pull request by asavioliMSFT**
It is not documented that in place upgrade on secondary replicas will leave databases in Synchronized / In Recovery state. Customers may think there is an issue, when they just need to perform a manual failover to finish the process

### CODE REVIEW

**Comment by Court72** [comment]
@MashaMSFT

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/database-engine/availability-groups/windows/upgrading-always-on-availability-group-replica-instances.md [hunk 1]
```diff
@@ -95,6 +95,11 @@ If necessary, you can perform an extra manual failover to return the AG to its o
 >
 > If `REQUIRED_SYNCHRONIZED_SECONDARIES_TO_COMMIT` is set to either `1` or `2`, the Primary replica may be unavailable for read/writes when a corresponding number of sync secondary replicas are not available during the update process.
 
+> [!NOTE]  
+> In case of Upgrade in-place on a secondary replica, which upgrades the instance to a newer version, the databases inside the AG will remain on state Synchronized / In Recovery. After a planned manual failover on the AG, the recovery will finish and the databases will be upgraded.
```
Comment (MashaMSFT) [inline docs/database-engine/availability-groups/windows/upgrading-always-on-availability-group-replica-instances.md]:
```suggestion
> When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.   
```

Comment (MashaMSFT) [reply docs/database-engine/availability-groups/windows/upgrading-always-on-availability-group-replica-instances.md]: @asavioliMSFT please commit the suggestion if you agree my changes haven't technically changed the meaning of your addition. thanks!

Comment (asavioliMSFT) [reply docs/database-engine/availability-groups/windows/upgrading-always-on-availability-group-replica-instances.md]: Thanks @MashaMSFT. I agree with your suggestion to change the text

**Comment by asavioliMSFT** [comment]
good afternoon @MashaMSFT. Anything else I should do in this PR or since I've already approved your changes everything that is required is done?

thanks

**Comment by MashaMSFT** [comment]
Hi @asavioliMSFT , thanks for making those changes! I'm just waiting for the PG to approve the change :) and then I can sign off on it.

Thanks!
Masha from the SQL Docs team

File: docs/database-engine/availability-groups/windows/upgrading-always-on-availability-group-replica-instances.md [hunk 2]
```diff
@@ -95,6 +95,11 @@ If necessary, you can perform an extra manual failover to return the AG to its o
 >
 > If `REQUIRED_SYNCHRONIZED_SECONDARIES_TO_COMMIT` is set to either `1` or `2`, the Primary replica may be unavailable for read/writes when a corresponding number of sync secondary replicas are not available during the update process.
 
+> [!NOTE]  
+> When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.
```
Comment (MashaMSFT) [inline docs/database-engine/availability-groups/windows/upgrading-always-on-availability-group-replica-instances.md]:
```suggestion
> When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronizing / In recovery** or **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.  An upgraded primary replica can no longer ship logs to any lower version secondary replica and data movement stops and no automatic failover can occur for that replica, and your availability databases are vulnerable to data loss. After you upgrade the old primary, you may need to resume synchronization.
```

File: docs/database-engine/availability-groups/windows/upgrading-always-on-availability-group-replica-instances.md [hunk 3]
```diff
@@ -95,6 +95,11 @@ If necessary, you can perform an extra manual failover to return the AG to its o
 >
 > If `REQUIRED_SYNCHRONIZED_SECONDARIES_TO_COMMIT` is set to either `1` or `2`, the Primary replica may be unavailable for read/writes when a corresponding number of sync secondary replicas are not available during the update process.
 
+> [!NOTE]  
+> When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronizing / In recovery** or **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.  An upgraded primary replica can no longer ship logs to any lower version secondary replica and data movement stops and no automatic failover can occur for that replica, and your availability databases are vulnerable to data loss. After you upgrade the old primary, you may need to resume synchronization.
```
Comment (MashaMSFT) [inline docs/database-engine/availability-groups/windows/upgrading-always-on-availability-group-replica-instances.md]:
```suggestion
> When you perform an in-place upgrade of a secondary replica to a newer version of SQL Server, the database inside the availability group remains in the **Synchronizing / In recovery** or **Synchronized / In Recovery** state until the availability group is manually failed over, which finishes the recovery and upgrades the database.  An upgraded primary replica can no longer ship logs to any lower version secondary replica and data movement stops and no automatic failover can occur for that replica, and your availability databases are vulnerable to data loss. After you upgrade the old primary, you may need to resume synchronization. It is recommended to upgrade all secondary replicas before failing over to a replica with the new version.  That way you have the option of doing a failover after the database(es) are upgraded to the new format.
```

**Comment by MashaMSFT** [comment]
#sign-off

**Pull request merged**